### PR TITLE
Supports Laravel 11

### PR DESCRIPTION
**Pull request by crynobone**

### CODE REVIEW

**Comment by freekmurze** [comment]
Nice! Let me know when I can review this one!

**Comment by crynobone** [comment]
Test failing and depends on https://github.com/symfony/symfony/pull/53521

File: tests/__snapshots__/CacheTest__it_can_detect_when_the_cache_is_missed__1.json [hunk 1]
```diff
@@ -6,9 +6,8 @@
                 "type": "table",
                 "content": {
                     "values": {
-                        "Event": "<code>Hit<\/code>",
-                        "Key": "cached-key",
-                        "Value": "cached-value"
+                        "Event": "<code>Missed<\/code>",
+                        "Key": "cached-key"
```
Comment (crynobone) [inline tests/__snapshots__/CacheTest__it_can_detect_when_the_cache_is_missed__1.json:10]: Correctly handle the assertion as it was incorrectly showing `Hit` when it should have been `Missed` due to defaulting to `file` cache driver.

File: .github/workflows/phpstan.yml [hunk 2]
```diff
@@ -12,7 +12,7 @@ jobs:
       - name: Setup PHP
         uses: shivammathur/setup-php@2.29.0
         with:
-          php-version: '8.0'
+          php-version: 8.3
```
Comment (crynobone) [inline .github/workflows/phpstan.yml:15]: I believe `phpstan` should be tested on the latest stable version.

**Comment by crynobone** [comment]
@freekmurze this is ready.

**Comment by freekmurze** [comment]
Thank you!

**Pull request merged**